### PR TITLE
Investigate doing android install and pipeline build of s2i images in…

### PR DESCRIPTION
**Pull request by luigizuccarelli**
… openshift

Motivation:
Install the android-sdk without breaking the accepting of the license agreement, create an s2i image with the installed sdk and push
the final slave image to the internal openshift registry avoiding the push of rather large (2G+) s2i images

Result:
The POC is feasable but some cleaning up wrt permissions and hostPath mounts

JIRA:
Refer to https://issues.jboss.org/browse/AGDIGGER-88

### CODE REVIEW

**Comment by wtrocki** [comment]
@luigizuccarelli - Checked this PR. Apart from small script duplication everything seems to be ok.
Basic cleanup required. Leaving this open once you would get back.

File: openshift/sdk-pv.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (wtrocki) [inline openshift/sdk-pv.json]: -1 for having this. We should recommend users to use oc-cluster wrapper:
https://github.com/openshift-evangelists/oc-cluster-wrapper

**Comment by matzew** [comment]
+1 on @wtrocki taking care

**Comment by wtrocki** [comment]
I always take care :)

**Comment by wtrocki** [comment]
Closing this PR as we would need to revise our target based on investigations. 
This PR contains useful bits and we going to use it in the future, but we cannot merge it as it is for the moment. 

Open questions that would need to talk about it:
- Using deployment config vs using build config.
- How to ask users to agree for the questions
- How we integrate this into ansible 

This work proves that we can do it and we would probably integrate current scripts into ansible.
After talking with the team I got impression that using s2i may be really confusing here. 

@matzew - let's do some developer sync on this one to make sure we all going into good direction. As @luigizuccarelli is going back to core team we would need to have wider discussion about it to make sure that entire team would be able to maintain it later.

**Comment by wtrocki** [comment]
Forgot to mention. We can always reopen and clean up this PR later.

**Pull request closed without merging**